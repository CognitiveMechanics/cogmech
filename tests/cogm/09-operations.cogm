
O(a, b) -> <a, b>

: O("a", "b")
: O("test", O("a", "b"))