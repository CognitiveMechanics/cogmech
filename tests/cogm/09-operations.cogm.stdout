<"a", "b">
<
  "test",
  <"a", "b">
>
